ow_lander/src/ow_lander/actions.py

```diff
@@ -1042,6 +1043,14 @@ def execute_action(self, goal):
 
     self._pub_trigger.publish()
 
+    # apply power usage for 2 cameras
```

Comment:
I think we should discuss, with @chetankul, whether our stereo camera should be treated as one entity or two.  I'm not sure which his power estimate is based on.  The two cameras are always triggered together, regardless of whether a left or right image capture is commanded.

Reply:
Sure. Bare in mind that at present this value makes a very minor difference in any case due to [OW-1239](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-1239). Also I actually based how camera power works off of comments made by Larry Edwards that were quoted in the [Jira ticket](https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-701).

Reply:
Will make this a single light system

Reply:
Done